virttest/qemu_monitor.py

```diff
@@ -2508,6 +2519,26 @@ def blockdev_change_medium(self, dev_id, filename, fmt=None, mode=None):
             args["read-only-mode"] = mode
         return self.cmd(cmd, args)
 
+    def blockdev_create(self, driver="file", size=0, job_id=None, **options):
+        """
+        Create block device image file by qemu
+
+        :param driver:  driver type,eg, file, qcow2, host_device and so on
+        :param size: image file size
+        :param job_id: job ID string
+        :param options: extra options not listed
+        :return: block job ID string
+        """
+        cmd = "blockdev-create"
+        self.verify_supported_cmd(cmd)
+        if not job_id:
+            job_id = utils_misc.generate_random_id()
+        mandatory_options = {"driver": driver, "size": size}
+        options.update(mandatory_options)
+        arguments = {"options": options, "job-id": job_id}
+        self.cmd("blockdev-create", arguments)
```

Comment:
Doesn't need to return this response back?

Reply:
No, job ID is enough.

Reply:
That would be inconsistent with other block job actions. I'd suggest making it return `self.cmd` here, and introducing a wrapper to return the job-id, any concern about that?